src/types.db

```diff
@@ -176,7 +179,13 @@ response_code		value:GAUGE:0:U
 route_etx		value:GAUGE:0:U
 route_metric		value:GAUGE:0:U
 routes			value:GAUGE:0:U
+sem_used_arrays    value:GAUGE:0:65535
+sem_used    value:GAUGE:0:65535
 serial_octets		rx:DERIVE:0:U, tx:DERIVE:0:U
+shm_bytes_rss    value:GAUGE:0:65535
+shm_bytes_swapped  value:GAUGE:0:65535
+shm_bytes_total    value:GAUGE:0:65535
+shm_segments    value:GAUGE:0:65535
```

Comment:
@ajdiaz we're trying to avoid infinite multiplication of the amount of entries in types.db. Would it be possible in some of the cases above to use type-instances to existing types (or to a reduced amount of new types) ? Thanks !